client/failover.go

```diff
@@ -0,0 +1,91 @@
+package client
+
+import (
+	"context"
+	"time"
+
+	"github.com/drand/drand/beacon"
+	"github.com/drand/drand/key"
+	"github.com/drand/drand/log"
+)
+
+const defaultFailoverGracePeriod = time.Second * 5
```

Comment:
worth defaulting this in relation to the group period?

Reply:
https://github.com/drand/drand/pull/383